release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -36,6 +38,17 @@ class CreateReleaseSubstepsState extends State<CreateReleaseSubsteps> {
   // Initialize a public state so it could be accessed in the test file.
   @visibleForTesting
   late Map<String, String?> releaseData = <String, String?>{};
+  @visibleForTesting
+  List<bool> isEachInputValid = List<bool>.filled(CreateReleaseSubsteps.substepTitles.length, false);
```

Comment:
what about using a map here to store the bools? an array where the indices are semantically meaningful makes the code harder to read and refactoring more difficult.